plugins/woocommerce/tests/legacy/unit-tests/rest-api/Helpers/OrderHelper.php

```diff
@@ -332,7 +333,7 @@ public static function create_complex_data_store_order( $data_store = null ) {
 		$order->set_billing_phone( '555-32123' );
 
 		$payment_gateways = WC()->payment_gateways->payment_gateways();
-		$order->set_payment_method( $payment_gateways['bacs'] );
+		$order->set_payment_method( $payment_gateways[ WC_Gateway_BACS::class ] );
```

Comment:
```suggestion
		$order->set_payment_method( $payment_gateways[ WC_Gateway_BACS::ID ] );
```

I think we intend to use the ID and not the class name here, right? This should a set of failing tests.